css/main.css

```diff
@@ -67,7 +67,26 @@ a.disabled {
     .welcome-pane .panel-body {
         min-height: 100px;
     }
+    /* #373  begin -----------*/
+    .log-workspace-panel {
+        width: 200px;
+    }
+    .dropdown-menu{
+        width:200px;
+    }
+    .workspace-selector-title {
+        width: 130px   
+    }
+    /* #373 end---------------*/
+}
+
+ /* #373  begin -----------*/
```

Comment:
Please remove these comments - users wanting to see change contained in a changeset can use `git blame`.